src/utils/hact/work-order-complete/close-job.test.js

```diff
@@ -15,15 +15,74 @@ describe('buildCloseWorkOrderData', () => {
       },
       jobStatusUpdates: [
         {
-          typeCode: 0,
+          typeCode: '0',
           otherType: 'complete',
           comments: `Work order closed - ${notes}`,
           eventTime: completionDate,
         },
       ],
     }
 
-    const response = buildCloseWorkOrderData(completionDate, notes, reference)
+    const response = buildCloseWorkOrderData(
+      completionDate,
+      notes,
+      reference,
+      'Job Completed'
+    )
+    expect(response).toEqual(CloseWorkOrderFormData)
+  })
+
+  //typeCode 0 for Job Completed
+  it('builds the ScheduleRepair form data to post to the Repairs API with typeCode 0', async () => {
+    const CloseWorkOrderFormData = {
+      workOrderReference: {
+        id: reference,
+        description: '',
+        allocatedBy: '',
+      },
+      jobStatusUpdates: [
+        {
+          typeCode: '0',
+          otherType: 'complete',
+          comments: `Work order closed - ${notes}`,
```

Comment:
I think the card requires us to add a prefix like `Work Order Closed - Completed` to this.

Reply:
the same as with No Access 
when I talked with Kyle and Josh they said that b/e will do it 
however, I can ask them about that tomorrow. Maybe will be faster to do it on our side